integration/integration_test.go

```diff
@@ -186,6 +190,106 @@ func TestDownloadSyncthing(t *testing.T) {
 	}
 }
 
+func TestStacks(t *testing.T) {
```

Comment:
Can you put it in a separate file in the same folder for clarity? (Make sure to copy the tags defined at the top of the file)